apps/web-client/src/lib/db/cr-sqlite/customers.ts

```diff
@@ -30,11 +30,38 @@
 
 import type { DB, Customer, DBCustomerOrderLine, CustomerOrderLine, BookLine } from "./types";
 
+/**
+ * Retrieves all customers from the database.
+ * Returns basic customer information ordered by ID.
+ *
+ * @param {DB} db - Database connection
+ * @returns {Promise<Customer[]>} Array of customers with:
+ *   - id: Unique identifier
+ *   - fullname: Customer's full name (optional)
+ *   - email: Contact email (optional)
+ *   - updatedAt: Last modification timestamp
+ *   - deposit: Current deposit amount (optional)
```

Comment:
I think we don't need to specify what's "inside" the types we're using.

```suggestion
 * @returns {Promise<Customer[]>} Array of customers
```